stable/artifactory-ha/values.yaml

```diff
@@ -56,6 +56,27 @@ ingress:
   #   hosts:
   #     - artifactory.domain.example
 
+networkpolicy:
+  # Allows all ingress and egress
+  - name: artifactory
+    podSelector:
+      matchLabels:
+        app: artifactory-ha
+    egress:
+    - {}
+    ingress:
+    - {}
+  # Uncomment to allow communicate to postgresql (if postgres.enabled is true)
```

Comment:
Shouldn't it be "Uncomment to allow only artifactory pods to communicate with postgresql (if postgres.enabled is true)"

Reply:
Done